src/vue/backend/src/views/guided-setup/Listings.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+    <GuidedSetupContentCard
+        :cardTitle="cardTitle"
+        :steps="guidedSetupSteps"
```

Comment:
The steps are not displaying their icons

Reply:
yeah, looks like the sprite got removed from `page` so I'll add it back in.